templates/ios/Example/Podfile

```diff
@@ -10,3 +10,27 @@ target '${POD_NAME}_Tests', :exclusive => true do
 
   ${INCLUDED_PODS}
 end
+
+# Add ${POD_NAME} target to build step of the example project
+# so that we get code coverage reports for ${POD_NAME}
+post_install do |installer|
+  # get ${POD_NAME} target from Pods project
+  pods_project = installer.pods_project
+  pods_targets = pods_project.targets
+  pod_target = pods_targets.select { |target| target.name == "${POD_NAME}" }.uniq.first
```

Comment:
Thanks. This is my first non-Podfile Ruby code. i-have-no-idea-what-im-doing.jpg